src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -358,23 +370,139 @@ def setClean(self, wflow):
         wflow['IsClean'] = self._checkClean(wflow)
         return wflow
 
+    def _checkLogDBClean(self, wflow):
```

Comment:
I would call it `isLogDBClean`. The docstring needs some working IMO.

Reply:
Ok, sorry again. I will need some more info about what exactly do you think needs to be worked on in the docstring. I will try to make it as clear as possible. 
About the name: I was following a pattern here and all private methods that are not supposed to be called independently are returning a `bool` variable instead of a `wflow` object and all of them I was prefixing them with the same substring `_check....` But if you think I should break the pattern I will rename it.

Reply:
In general I tend to create a separate method when a separate call just to the check method is expected to be made somewhere else into the code. In all other cases, I do agree with you that having the logic enclosed inside the function that is about to change the flag status itself is the best. I will see what I can do with the `_checkLogDB` method.

Reply:
Thanks!

Reply:
Marking it as resolved! Thanks